unidist/core/backends/mpi/core/controller/common.py

```diff
@@ -115,6 +106,34 @@ def release_rank(self, rank):
         )
 
 
+def get_data(comm, owner_rank):
```

Comment:
```suggestion
def receive_request(comm, owner_rank):
```
It seems this should be in communication,py. Also, add a docstring for the function.

Reply:
This method can not be placed in communication.py, because this logic requires references to LocalObjectStore and SharedObjectStore.

Reply:
"receive_request" is not understandable name, because the main task of this func is receiving data from another MPI process using shared memory or direct communiction, depending on package type.
Also common module includes the "push_data" func, and this func is opposit for it.

Reply:
If this is the opposite to push_data, then it might be better to name this function as `pull_data`. What do you think?

Reply:
In my opinion it will be ok.

Reply:
Good, let's change it then.